examples/ShowPlayer.cpp

```diff
@@ -85,55 +92,84 @@ int ShowPlayer::Playback(unsigned int iterations,
   return ola::EXIT_OK;
 }
 
-void ShowPlayer::SendNextFrame() {
-  DmxBuffer buffer;
-  unsigned int universe;
-  ShowLoader::State state = m_loader.NextFrame(&universe, &buffer);
-  switch (state) {
-    case ShowLoader::END_OF_FILE:
-      HandleEndOfFile();
-      return;
-    case ShowLoader::INVALID_LINE:
-      m_client.GetSelectServer()->Terminate();
-      return;
-    default:
-      {}
-  }
 
-  state = RegisterNextTimeout();
+/**
+ * Start playback
+ */
+void ShowPlayer::Start() {
+  SeekTo(m_start);
+}
+
 
-  OLA_INFO << "Universe: " << universe << ": " << buffer.ToString();
-  ola::client::SendDMXArgs args;
-  m_client.GetClient()->SendDMX(universe, buffer, args);
-
-  switch (state) {
-    case ShowLoader::END_OF_FILE:
-      HandleEndOfFile();
-      return;
-    case ShowLoader::INVALID_LINE:
-      m_client.GetSelectServer()->Terminate();
-      return;
-    default:
-      {}
+/**
+ * Seek to @p time in the show file
+ * @param time the time (in milliseconds) to seek to
+ */
+void ShowPlayer::SeekTo(const unsigned int time) {
```

Comment:
Time is a used function:
http://www.cplusplus.com/reference/ctime/time/

So it would be good to use another variable name, perhaps seek_time?